tests/clock/jobs/deactivateRooms.js

```diff
@@ -23,7 +23,7 @@ const ZERO = 0;
 const ONE = 1;
 const MOON_LANDING = '1969-07-20T20:18:00+00:00';
 
-describe('tests/clock/jobs/deactivateRooms.js >', () => {
+describe.skip('tests/clock/jobs/deactivateRooms.js >', () => {
```

Comment:
No reason to skip if it's still possible to enable via toggle.

Reply:
Nice @jgraff2, let me work on that. Thx